src/components/RegisterComponent/RegisterComponent.jsx

```diff
@@ -10,52 +10,52 @@ class RegisterComponent extends Component {
       name: "",
       email: "",
       password: "",
-      "confirm password": "",
-      height: 200,
-      weight: 75,
-      activity_level_id: 1,
-      birth_date: "1999-01-01",
-      gender_id: 1,
-      user_tier_id: 1,
-      goal_id: 1,
-      formErrors: {
-        name: "",
-        email: "",
-        password: "",
-        "confirm password": "",
-        height: "",
-        weight: "",
-        activity_level: "",
-        birth_date: ""
-      }
+      confirm_password: ""
     };
   }
-
+  
   handleInput = event => {
     const { value, name } = event.target;
     const state = { ...this.state };
     state[name] = value;
     this.setState(state);
   };
 
-  handleRegister = e => {
-    e.preventDefault();
-
-    let formData = { ...this.state };
-    delete formData.formErrors;
-    delete formData["confirm password"];
-    this.props.dispatchRegister(formData);
-    return this.props.isRegistered();
-  };
-
   handleLoginClick = () => {
     return this.props.isRegistered();
   };
 
+  handleContinueClick = (e) => {
+    e.preventDefault();
+    if (!this.state.name || !this.state.email || !this.state.password){
+      alert("Please fill in all fields");
+      return false;
+    }
+    if (!this.validateEmail(this.state.email)){
+      alert("Please enter a valid email address");
+      return false;
+    }
+    if (this.state.password !== this.state.confirm_password){
+      alert("Passwords do not match.");
+      return false;
+    }
+    if (this.state.password.length < 6){
+      alert("Password must be at least 6 characters long.");
+      return false;
+    }
+    return this.props.takingSurvey(this.state);
+    
+  }
```

Comment:
Thinking about it now, this component will also need a function that calls to the database to check if the email is already registered.  It's not mission critical right now, so if you don't want to deal with it, it should be fine, but could you add it to the trello board for debugging if you decide not to?

Reply:
Ok I added the email check and fixed up the small bug. Thanks for catching that.